src/components/Cart/CartItem.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect, useContext } from "react";
+import { CartContext } from "../../util/context";
+import {
+  Box,
+  Flex,
+  Image,
+  Heading,
+  Spacer,
+  Text,
+  CloseButton,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  NumberIncrementStepper,
+  NumberDecrementStepper,
+} from "@chakra-ui/react";
+
+const CartItem = ({
+  imageURL,
+  unitPrice,
+  title,
+  description,
+  handleClick,
+  id,
+  itemQuantity,
+}) => {
+  const [quantity, setQuantity] = useState(itemQuantity);
+  const [totalprice, setTotalPrice] = useState(0);
+  const { handleAddQuantity, handleRemoveItemFromCart } =
+    useContext(CartContext);
+
+  const HandleQuantityChange = (value) => {
+    setQuantity(parseInt(value));
+  };
+
+  useEffect(() => {
+    setTotalPrice((quantity * parseFloat(unitPrice)).toFixed(2));
+    handleAddQuantity(id, quantity);
+  }, [quantity]);
+
+  return (
+    <Flex
+      maxW='full'
+      borderWidth='1px'
+      borderRadius='16px'
+      p={4}
+      mb={6}
+      align='center'
+    >
+      <Image
+        src={imageURL}
+        alt='image'
```

Comment:
In regards to best practices, it is best to include a descriptive alt tag. It can be as simple as "Image of 2022 MacBook Air" (including the item's title is good enough).